chapters/develop-organize/drills/reading/README.md

```diff
@@ -38,14 +35,59 @@ There can be essays / open-ended questions that require a personal take on a top
+Examples of such answers multiple-choice questions, predefined words, numbers, etc.
+Questions are the best candidates for automatic grading due to the predictability of their answers.
+
+1. [Tasks](./tasks.md) on the other hand require students / trainees to provide a **solution**, such as a proof, the steps how they reached the answer or an essay.
+While difficult to automate with regards to grading, tasks are a more profound type of drill that specifically targets the students' reasoning.
+This is why grading them should be done by humans (most of the time).
+
+The descriptions of the two types of drills outlined above are not exhaustive.
+There are in fact countless types of drills that may be used.
+For a more in-depth look, you may consult the various quiz deployment platforms in existence.
+In our opinion, two strong starting points are [Blackboard](https://help.blackboard.com/Learn/Instructor/Ultra/Tests_Pools_Surveys/Question_Types) and [Moodle](https://docs.moodle.org/402/en/Question_types)
+
+## Storage
+
+We propose that each drill be stored in a separate folder together with all supporting files, such as images, solutions, code, etc.
+This allows great flexibility when importing, exporting or extracting drills, as all files related to a particular one are grouped.
+
+For a more detailed analysis of the contents of each drill type's folders, check the corresponding sections regarding [Questions](./questions.md#storage) and [Tasks](./tasks.md#storage).
+
+## Use Cases
+
+Drills may become part of any teaching stage:
 
-TODO: mention flexibility of storing the different types of drill items
+- During lectures, **questions** may be used for quick quizzes to keep the audience engaged.
 
-TODO: mention use cases: lab, lecture, assignments, exams, self study (basically everywhere)
+- During labs **questions** may once again be grouped into quizzes while **tasks** make up the exercises that students must solve.
 
-TODO: mention roles: learn / practice, self-assess, actual evaluation
+- Assignments can sometimes consist of one or more drills.
+This applies to both **questions** that can be employed as quizzes and **tasks**, such as essays or reports.
 
-Sample question types: [`Blackboard`](https://help.blackboard.com/Learn/Instructor/Ultra/Tests_Pools_Surveys/Question_Types), [Moodle](https://docs.moodle.org/402/en/Question_types)
+- For exams, drills are useful for practical exams while questions are for quiz exams.
 
-link different types of drills as `.md` files in the current directory: [questions](questions.md) and [tasks](tasks.md)
+As mentioned in the [section above](#use-cases), drills used in lectures and labs are more suited as learning / practice items.
+In particular, during labs and sometimes assignments, questions may be a tool for students to evaluate themselves, without intervention from the teacher, while exams mmay use drills for actual evaluation.
+In this case, tasks become relevant for teacher-graded evaluation.
```

Comment:
The classical ["this declaration does not declare anything"](https://stackoverflow.com/questions/21610637/declaration-does-not-declare-anything-warning).